src/main/zlatopolsky/chapter10/Ch1N10_51.java

```diff
@@ -0,0 +1,27 @@
+package zlatopolsky.chapter10;
+
+public class Ch1N10_51 {
+
+    public int recursionExpressionA(int n) {
```

Comment:
It should print the results like:
5
4
3
2
1

Reply:
yes it error, i'm fix the method

Reply:
fix